### DISCUSSION TITLE
Support for event location

### DISCUSSION THREAD

**Discussion by konata-san** [davish/obsidian-full-calendar#413] [Feature Requests]
It would be nice to be able to specify the location of an event.
- This could possibly also include a field for travel time, with the option to auto calculate travel time based on the address using something like google maps.

**Comment by Yodo9000** [comment]
Agree. Imported icals can also contain location data. The Google Calendar shows this, but in this plugin I just get the notification "Read-only events cannot be modified". It would be nice if this plugin could also import/show this location data, it would make finding my classes a lot easier ;-).
